doc/cephfs/administration.rst

```diff
@@ -49,7 +49,7 @@ Settings
 
     fs set <fs name> max_file_size <size in bytes>
 
-CephFS has a configurable maximum file size, and it's 1TB by default.
+CephFS has a configurable maximum file size, and it is 1TB by default.
```

Comment:
This was from one of our writers in the Brno. It is to be avoided per the IBM style guide. It also makes it easier to translate.